local-install.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# Exit on error, run verbose
+set -ev
+
+# Run from a script to make sure Travis CI handles failures properly:
+# http://steven.casagrande.io/articles/travis-ci-and-if-statements/
+
+if   [ $TRAVIS_PYTHON_VERSION == 2.6 ]; then python 2.6/get-pip.py;
```

Comment:
I think, these two scripts could be turned into one: they are 90% copy-paste with just one variable thing.

Reply:
you can have a function which either `cat`s script or `wget`s it

Reply:
@hugovk 

Smth like `helpers.sh`:

```shell
function cat_get_pip() {
  local version="$1"
  local path="$version/get-pip.py"
  local src="$2"
  local cmd=cat
  if [[  "$src" == "remote" ]]
  then
    path="https://bootstrap.pypa.io/$path"
    cmd='wget -O - '
  elif [[ "$src" != "local" ]]
    >&2 echo Wrong source argument: $src
    exit 1
  fi
  
  $cmd $path
}
```

and then have:
```yaml
before_script:
- source helpers.sh
script:
- cat_get_pip "$TRAVIS_PYTHON_VERSION" "$SCRIPT_SRC" | python -

matrix:
  global:
  - SCRIPT_SRC=local
  - SCRIPT_SRC=remote
```